### PR TITLE
Added RightSidebar Kodi icon and  hover effect on progress slider

### PR DESCRIPTION
**Pull request by developingright**
Hi @razzeee ,
I have added Kodi icon to the RightSidebar and hover effect on the progress slider like in Chorus-2.

Name             |  RideSidebar Kodi button  |  Progress Slider
:-------------------------:|:-------------------------: |:-------------------------:
Chorus-2 | Screenshot from 2022-04-02 14-50-39 | Screenshot from 2022-04-02 14-59-54

****
Name             |  Before making changes  |  After making changes
:-------------------------:|:-------------------------: |:-------------------------:
Elm-chorus -> RightSidebar Kodi button  |  161093468-caf63bb8-6d88-42d6-9b6a-e405bcc38f67 (1) |  Screenshot from 2022-04-02 14-52-23
Elm-Chorus -> Progress Slider | Screenshot from 2022-04-02 15-05-56 | Screenshot from 2022-04-02 15-08-02

### CODE REVIEW

**Review by razzeee — CHANGES_REQUESTED** [review]

File: src/Components/RightSidebar.elm [hunk 1]
```diff
@@ -17,7 +18,17 @@ view { showRightSidebarMenu, showRightSidebarMenuMsg, clearPlaylistMsg, refreshP
...
             [ height (px panelHeight), width (px 400), Background.color Colors.playlistHeaderBackground, alignRight, htmlAttribute <| Html.Attributes.style "pointer-events" "all" ]
             [ row [ width fill ]
-                [ Input.button [ Background.color Colors.backgroundKodi, height (px 50), width (px 100), padding 8 ] { onPress = Just rightSidebarMsg, label = Element.text "Kodi" }
+                [ Input.button [ Background.color Colors.backgroundKodi, height (px 50), width (px 100), padding 8 ]
+                    { onPress = Just rightSidebarMsg
+                    , label =
+                        Element.row []
+                            [ image [ width (px 15), height (px 15), paddingXY 15 0 ]
+                                { description = ""
+                                , src = "https://raw.githubusercontent.com/ionif/elm-chorus/master/logo.png"
```
[the remark this answers is not in the document]
Comment (developingright) [reply src/Components/RightSidebar.elm]: Hi @razzeee , 
	I have made changes to the code, now the images which is being used in the rightsidebar is local.

**Comment by developingright** [comment]
@razzeee ,
Since Elm doesn’t support autoloading, one way to achieve this was to use the development server to serve the static files (for ex: “http:localhost:1234/images/logo.png” as image src). Since we are using parcel for running the development server, I have installed a parcel plugin “[parcel-plugin-static-files-copy](https://www.npmjs.com/package/parcel-plugin-static-files-copy)” and created an image folder in public folder which is mapped to the server. Now, I will create a PR to make all of the static files link local which is an already existing [issue](https://github.com/xbmc/elm-chorus/issues/36).

**Comment by razzeee** [comment]
This should be a dev dependency, not a dependency.

It also seems to be a parcel 1 plugin, but we're using parcel 2

**Comment by developingright** [comment]
Hi @razzeee,
I have updated the package.json file and updated the plugin which uses parcel2 “[parcel-reporter-static-files-copy](https://github.com/elwin013/parcel-reporter-static-files-copy)” for serving the static files.

**Comment by razzeee** [comment]
This is conflicting now, you probably need to rebase

**Comment by developingright** [comment]
Hi @razzeee,
I have rebased the code and resolved the conflicts.

**Comment by razzeee** [comment]
I don't think you did that correctly :)
There are a bunch of unrelated commits

**Pull request closed without merging**